clients/cobol-lsp-vscode-extension/src/constants.ts

```diff
@@ -26,6 +26,7 @@ export const DOWNLOAD_QUEUE_LOCKED_ERROR_MSG = `Invalid credentials for profile:
 export const UNLOCK_DOWNLOAD_QUEUE_MSG = "Unblock and retry";
 export const INVALID_CREDENTIALS_ERROR_MSG = `Incorrect credentials in Zowe profile ${PROFILE_NAME_PLACEHOLDER}.`;
 export const PROCESS_DOWNLOAD_ERROR_MSG = "Some copybooks could not be located. Ensure your configuration contains correct paths to copybooks, including nested copybooks. Missing copybooks: ";
+export const PROVIDE_PROFILE_MSG = "Please specify a Zowe Explorer profile for copybook downloading."
```

Comment:
@zeibura when no profile is configured.

Reply:
"Please specify a Zowe Explorer profile to download copybooks from the mainframe"

Reply:
updated